### PR TITLE
PS-1154 ABS workspace for tables

### PR DESCRIPTION
**Pull request by pivnicek**
Just doing tables for now.  Files will be in the nest PR

### CODE REVIEW

File: src/Writer/TableWriter.php [hunk 1]
```diff
@@ -607,6 +614,10 @@ private function convertStagingToStorageApiBackend($stagingStorageOutput)
                 return WorkspaceProviderInterface::TYPE_REDSHIFT;
             case self::STAGING_SYNAPSE:
                 return WorkspaceProviderInterface::TYPE_SYNAPSE;
+            case self::STAGING_ABS:
+                // TODO: when input mapping is released use the new type from there
```
Comment (JakubMatejka) [inline src/Writer/TableWriter.php:618]: Wouldn't it be better to wait for IM then? ;o)

Comment (pivnicek) [reply src/Writer/TableWriter.php:618]: Yes! :-)

Comment (odinuv) [reply src/Writer/TableWriter.php:618]: IM released, so go ahead :)

**Comment by pivnicek** [comment]
Ok, we're greeen here now :-)

File: composer.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
         "ext-json": "*"
     },
     "scripts": {
-        "tests": "phpunit",
+        "tests": "phpunit --filter testAbsTableOutputMapping",
```
Comment (JakubMatejka) [inline composer.json:40]: Really?

Comment (pivnicek) [reply composer.json:40]: tvl

Comment (pivnicek) [reply composer.json:40]: thanks

Comment (JakubMatejka) [reply composer.json]: I wasn't sure but it looked suspicious 😄

**Review by odinuv — APPROVED** [review]

**Pull request merged**